fhir-server/src/main/java/org/linuxforhealth/fhir/server/util/FHIRRestHelper.java

```diff
@@ -602,6 +602,19 @@ public FHIRRestOperationResponse doUpdateMeta(FHIRPersistenceEvent event, String
                         SingleResourceResult<? extends Resource> srr = doRead(type, id, false, null, false);
                         ior.setPrevResource(srr.getResource()); // might be null if resource is deleted
                         isDeleted = srr.isDeleted();
+                        
+                        // Check that the resource exists, unless the updateCreate feature is enabled.
+                        // No matches, id provided and doesn't already exist and updateCreate feature is not enabled : Should be rejected with error message.
+                        if (srr.getResource() == null && !persistence.isUpdateCreateEnabled()) {
+                            String msg = "Resource '" + type + "/" + id + "' not found.";
```

Comment:
if we move this comment to after the if statement it will be more consistent with the else if
```suggestion
                        if (srr.getResource() == null && !persistence.isUpdateCreateEnabled()) {
                            // No matches, id provided and doesn't already exist and updateCreate feature is not enabled : Should be rejected with error message.
                            String msg = "Resource '" + type + "/" + id + "' not found.";
```

Reply:
This has been fixed as suggested.